roles/migrationcontroller/tasks/main.yml

```diff
@@ -483,7 +517,7 @@
       namespace: "{{ meta.namespace }}"
       status:
         phase: Reconciled
-    when: olm_managed and reconciled
+    when: reconciled
```

Comment:
At least some versions of OCP3 don't have status sub-resources. We're doing this as a hacky way to allow the operator to work on OCP3 (back as far as 3.7) so we should leave it.